src/tasks/Microsoft.NET.Sdk.WebAssembly.Pack.Tasks/GenerateWasmBootJson.cs

```diff
@@ -138,6 +147,9 @@ public void WriteBootJson(Stream output, string entryAssemblyName)
             result.runtimeOptions = runtimeOptions.ToArray();
         }
 
+        var libraryInitializerOnRuntimeConfigLoaded = LibraryInitializerOnRuntimeConfigLoaded?.Select(s => s.GetMetadata("FullPath")).ToArray() ?? Array.Empty<string>();
+        var libraryInitializerOnRuntimeReady = LibraryInitializerOnRuntimeReady?.Select(s => s.GetMetadata("FullPath")).ToArray() ?? Array.Empty<string>();
```

Comment:
1. Use explicit types here, because the type isn't immediately obvious
2. name of the locals is too similar to the property, making it easy to mix things up.